Loop/Views/Presets/NewCustomPreset.swift

```diff
@@ -110,13 +110,31 @@ extension NewCustomPreset {
             targetRange: correctionRange,
             insulinNeedsScaleFactor: insulinMultiplier
         )
+        
+        let split = name.splitSymbolAndTitle()
+        var symbol: PresetSymbol? = nil
+        if let emoji = split.emoji {
+            symbol = .emoji(emoji)
+        }
 
         return TemporaryPreset(
-            symbol: "",
-            name: name,
+            symbol: symbol,
+            name: split.name,
             settings: settings,
             duration: overrideDuration,
             scheduleStartDate: startDate
         )
     }
 }
+
+private extension String {
+    func splitSymbolAndTitle() -> (emoji: String?, name: String) {
+        let trimmed = trimmingCharacters(in: .whitespaces)
+        if let first = trimmed.first, first.isEmoji {
```

Comment:
Integers (and maybe some other unexpected characters) return true for isEmoji

Reply:
Okay I've update this `isEmoji` helper to work on `Unicode.Scaler.isEmojiPresentation` instead of `Unicode.Scaler.isEmoji`. Seems to be like the exact thing we want.

https://developer.apple.com/documentation/swift/unicode/scalar/properties-swift.struct/isemojipresentation